### PR TITLE
added http DELETE method

### PR DESCRIPTION
**Pull request by wunderlink**
One line change. Bit more had to happen to the tests...basically just cloning the "get" tests for "delete".

### CODE REVIEW

File: jsonist.js [hunk 1]
```diff
@@ -108,3 +108,4 @@ function isRedirect (request, response) {
 module.exports.get  = makeMethod('GET'  , false)
 module.exports.post = makeMethod('POST' , true)
 module.exports.put  = makeMethod('PUT'  , true)
+module.exports.delete  = makeMethod('DELETE'  , false)
```
Comment (rvagg) [inline jsonist.js]: note the pattern here, the `=` are all lining up regardless of the length of the name and the comma and booleans are all lining up too, could you try and reformat these 4 lines so they line up again? `post` was previously the longest so set the padding but now `delete` is longer so will determine that.

File: test.js [hunk 2]
```diff
@@ -37,47 +37,49 @@ function testServer (serverResponse) {
 }
 
 
-test('fetch json doc', function (t) {
-  t.plan(7)
+'get delete'.split(' ').forEach(function (type) {
+  test(type+' fetch json doc', function (t) {
```
Comment (rvagg) [inline test.js]: whitespace please, ` + `

File: test.js [hunk 3]
```diff
@@ -37,47 +37,49 @@ function testServer (serverResponse) {
...
+      .on('request', function (req, data) {
+        t.equal(req.method, type.toUpperCase(), 'got ' + type + ' request')
+        t.equal(req.headers['accept'], 'application/json', 'got correct accept header')
+      })
+      .on('close', t.ok.bind(t, true, 'ended'))
+  })
 
-test('fetch non-json doc', function (t) {
-  t.plan(7)
+  test(type+' fetch non-json doc', function (t) {
```
Comment (rvagg) [inline test.js]: whitespace

**Comment by rvagg** [comment]
looks good except for a few minor whitespace issues

**Comment by wunderlink** [comment]
I think I got the whitespace matching existing patterns. Thanks for the quick response!

**Comment by rvagg** [comment]
thanks, merged @ 5aa95e9 and published as 1.3.0

**Comment by rvagg** [comment]
I mean 9197073596b23bd46119984606f00b499038bf5e & e0a2fd3b0f9804cedc9a7bc925b58bf088ad774e

**Comment by wunderlink** [comment]
Awesome! Thanks!

**Pull request closed without merging**